### DISCUSSION TITLE
Prediction values not reproducible after upgrading pyg to 2.2.0

### DISCUSSION THREAD

**Discussion by nestiank** [pyg-team/pytorch_geometric#6547] [Q&A]
Hello,

I used to use pyg 2.0.3 with pytorch 1.9.1 for my project.
After I upgrade pyg to 2.2.0 and pytorch to 1.13.1 and load the same model, it produces different prediction values.

I could not find any critical difference between those version changes, so please help!

Thanks.

**Comment by rusty1s** [comment]
Which operator/model are you using? Do you have a reproducible example?
